cmd/neofs-cli/modules/object/range.go

```diff
@@ -58,7 +58,7 @@ func getObjectRange(cmd *cobra.Command, _ []string) {
 
 	var out io.Writer
 
-	filename := cmd.Flag(fileFlag).Value.String()
+	filename := cmd.Flag(commonflags.FileFlag).Value.String()
```

Comment:
This also seems like a refactoring, could we have a separate commit?
Also these `--file` flags have different semantics, I think it is ok not to move them to `commonflags`.

Reply:
Changes were reverted.